src/pybind/mgr/dashboard/frontend/src/app/shared/api/mgr-module.service.ts

```diff
@@ -53,4 +53,13 @@ export class MgrModuleService {
   disable(module: string) {
     return this.http.post(`${this.url}/${module}/disable`, null);
   }
+
+  /**
+   * Get the Ceph Mgr module options.
+   * @param {string} module The name of the mgr module.
+   * @return {Observable<Object>}
+   */
+  getOptions(module: string) {
```

Comment:
Probably it's better to specify the return type in the function and let the doc generator to take that info from there 

```suggestion
  getOptions(module: string):Observable<Object> {
```

Reply:
Done to all methods that return an observable.